auth0/structure_auth0_connection.go

```diff
@@ -172,8 +182,9 @@ func expandConnection(d Data) *management.Connection {
 			c.Options = expandConnectionOptionsAuth0(d)
 		case management.ConnectionStrategyGoogleOAuth2:
 			c.Options = expandConnectionOptionsGoogleOAuth2(d)
+		case management.ConnectionStrategyApple:
+			c.Options = expandConnectionOptionsApple(d)
 		// case management.ConnectionStrategyFacebook
-		// 	management.ConnectionStrategyApple
```

Comment:
Can we remove this as well?

Reply:
Similar to the above comment, this line ~is~ was commented out (stub) in `origin/master` and I removed it as part of making the necessary changes. That's why it shows up (as red / removed) in the diff.